healthcare/api-client/dicom/dicom_stores.py

```diff
@@ -342,14 +415,29 @@ def parse_command_line_args():
         help='Specifies the output format. If the format is unspecified, the'
         'default functionality is to export to DICOM.')
 
+    parser.add_argument(
+        '--member',
+        default=None,
+        help='Member to add to IAM policy (e.g. "domain:example.com")')
+
+    parser.add_argument(
+        '--role',
+        default=None,
+        help='IAM Role to give to member (e.g. "roles/viewer")')
+
     command = parser.add_subparsers(dest='command')
 
     command.add_parser('create-dicom-store', help=create_dicom_store.__doc__)
     command.add_parser('delete-dicom-store', help=delete_dicom_store.__doc__)
     command.add_parser('get-dicom-store', help=get_dicom_store.__doc__)
     command.add_parser('list-dicom-stores', help=list_dicom_stores.__doc__)
     command.add_parser('patch-dicom-store', help=patch_dicom_store.__doc__)
-
+    command.add_parser(
+        'get_iam_policy',
```

Comment:
lint is not liking this line

Reply:
I'll thought I'd handled the lint issues. I'll clean them all up in about an hour.